Formula/python.rb

```diff
@@ -336,6 +352,10 @@ def sitecustomize
   end
 
   def caveats; <<-EOS.undent
+    This formula installs a python2 executable to #{HOMEBREW_PREFIX}/bin.
+    If you wish to have this formula's python executable in your PATH then add:
+      PATH="#{opt_libexec}/bin:$PATH"
```

Comment:
Should we say _where_ to add it (shell profile)? As was mentioned earlier in the conversation somewhere, people wanting this seem to be quite likely to be very new to all this, so it might be helpful to be more explicit here.